src/Provider/AkeneoAttributeDataProvider.php

```diff
@@ -46,7 +46,7 @@ public function getData(string $attributeCode, $attributeValues, string $locale,
 
         if (!$this->akeneoAttributePropertyProvider->isScopable($attributeCode) &&
             $this->akeneoAttributePropertyProvider->isLocalizable($attributeCode)) {
-            return $this->getByLocale($attributeCode, $attributeValues, $locale);
+            return $this->getByLocale($attributeCode, $attributeValues, $locale, $scope);
```

Comment:
All changes in this file do not seem to be true :thinking: because we get into this function when the field is not scopable. Maybe there is a problem elsewhere, we have to investigate more on that.